freelist_test.go

```diff
@@ -435,6 +435,11 @@ func newTestFreelist() *freelist {
 }
 
 func Test_freelist_hashmapGetFreePageIDs(t *testing.T) {
+	// Because we pre-create the freelist here, we'd fail the assertion that
+	// the computed count matches the freePagesCount.
+	reenable := common.DisableVerifications()
+	defer reenable()
```

Comment:
Let's update the `freePagesCount` instead of disabling the verification. Pls add line below inside the for loop.

```
f.freePagesCount += uint64(val)
```